indexer/Dockerfile

```diff
@@ -13,26 +13,23 @@ FROM ubuntu:20.04
 ARG TOKEN
 ENV GITHUB_TOKEN=$TOKEN
 
+ARG DEBIAN_FRONTEND=noninteractive
+
 RUN if [ -z "$GITHUB_TOKEN" ]; then \
       echo "GITHUB_TOKEN should be provided during build stage."; \
       exit -1; \
     fi;
 
-ENV TZ=Europe/Berlin
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
-
 # Install dependencies.
 RUN apt-get update; \
-    apt-get install -y git curl sudo python3 python3-requests p7zip-full unzip \
-                       python libc6-i386 lsb-release gnupg cron \
-                       software-properties-common; \
+    apt-get install -y git curl sudo python3 \
+                       python3-requests p7zip-full unzip \
+                       python libc6-i386 lsb-release gnupg cron; \
     apt-get dist-upgrade -y;
```

Comment:
do we still need `dist-upgrade` and `lsb-release` and `gnupg` ? can we just do `apt-get upgrade` ?